app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -190,6 +263,15 @@ class Scene extends React.Component {
 
   serialise(object) {
     return this.res.stringify(object, (key, value) => {
+      // serialise fails if it attempts to work on a function, even if it's a member of an object
+      // to prevent this from occuring, we need to check objects for their first member to see if it's a function
+      if (
+        value &&
+        typeof value === typeof object &&
+        typeof value[0] === typeof (() => {})
```

Comment:
can we check the `key` instead of the `typeof value[0]`?

Reply:
This code also works, if you think it would be better:
`if (key === 'sleepStart') {
return null;
      }`

Reply:
I would prefer to ignore all `key === 'events'` if possible